src/broker/cas_protocol.h

```diff
@@ -292,6 +292,8 @@ extern "C"
         (CAS_PROTO_MAKE_VER(CAS_PROTO_VER_MASK & (char)(VER)))
 #define CAS_PROTO_PACK_CURRENT_NET_VER      \
         CAS_PROTO_PACK_NET_VER(CURRENT_PROTOCOL)
+#define CAS_CCI_PROTO_PACK_CURRENT_NET_VER      \
+        CAS_PROTO_PACK_NET_VER(PROTOCOL_V7)
 
```

Comment:
How about moving this definition to 'cci_network.c' to reduce dependency with CUBRID Engine code.

Reply:
It looks the way to move the code location is good point.